documentation/src/main/asciidoc/user_guide/chapter-76-Functional_Map_API.adoc

```diff
@@ -0,0 +1,775 @@
+== Functional Map API
+
+Infinispan 8 introduces a new experimental API for interacting with your
+ data which takes advantage of the functional programming additions and
```

Comment:
I started when I was editing in the IDE, but didn't look too bad after IMO. I can revert it back.